esp/src/src-react/components/DaliAdmin.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { Pivot, PivotItem } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { pushUrl } from "../util/history";
+import { pivotItemStyle } from "../layouts/pivot";
+import { GetDFSCSV, GetDFSMap, GetDFSParents, GetLogicalFile, GetLogicalFilePart, GetProtectedList, GetValue } from "src/WsDali";
```

Comment:
It wasn't available at the time (I think it was only added about a week ago) but you could change this import to be 
`from "@hpcc/comms` and eliminate that src/WsDali file.